core/views.py

```diff
@@ -22,105 +22,141 @@
 import googlemaps
 from datetime import datetime
 import pprint
-#https://learndjango.com/tutorials/django-signup-tutorial   
+#https://learndjango.com/tutorials/django-signup-tutorial
 class SignUpView(generic.CreateView):
     form_class = UserCreationForm
     success_url = reverse_lazy("login")
     template_name = "registration/signup.html"
 
 def standardize_cms_name(cms_name_df):
-    cms_name_df = cms_name_df.str.lower()
-    cms_name_df = cms_name_df.str.replace('-', " ")
-    cms_name_df = cms_name_df.str.replace('/', " ")
-    return cms_name_df
+    return cms_name_df.str.lower().replace('-', " ").replace('/', " ")
+
+@timeit
+def load_data():
```

Comment:
might as well rename to load_summary_metrics to be more specific

Reply:
will do

Reply:
done